plugins/CoreAdminHome/templates/generalSettings.twig

```diff
@@ -129,6 +129,20 @@
                      autocomplete="off">
                 </div>
 
+                <div piwik-field uicontrol="text" name="mailFromAddress"
+                     ng-model="mailSettings.mailFromAddress"
+                     title="{{ 'General_SmtpFromAddress'|translate|e('html_attr') }}"
+                     value="{{ mail.noreply_email_address }}" inline-help="{{ 'General_UsuallyYourEmailAddress'|translate|e('html_attr') }}"
+                     autocomplete="off">
+                </div>
+
+                <div piwik-field uicontrol="text" name="mailFromName"
+                     ng-model="mailSettings.mailFromName"
+                     title="{{ 'General_SmtpFromName'|translate|e('html_attr') }}"
+                     value="{{ mail.noreply_email_name }}" inline-help="{{ 'General_NameShownInTheSenderColumn'|translate|e('html_attr') }}"
```

Comment:
I'm not 100% sure but it may need to be `{{ mail.noreply_email_name |e('html_attr') }}` to escape it re security